src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1113,6 +1113,10 @@ private function registerPropertyAccessConfiguration(array $config, ContainerBui
             ->replaceArgument(0, $config['magic_call'])
             ->replaceArgument(1, $config['throw_exception_on_invalid_index'])
         ;
+
+        if (isset($config['naming_strategy']) && $config['naming_strategy']) {
```

Comment:
perhaps add a `cannotBeEmpty()` in config, so only isset() is needed here.

Reply:
no need for isset() actually as the node does not seem to be removed if not set (hence will have an empty value). A `cannot be empty` would break BC IIRC.

Reply:
> A cannot be empty would break BC IIRC.

Why so? This is a new config node.

No real opinion about `isset($config['node'])` vs. `null !== $config['node']`.